testing/scala/APKBUILD

```diff
@@ -0,0 +1,111 @@
+# Contributor: norbjd
+# Maintainer:
+pkgname=scala
+pkgver=2.12.2
+pkgrel=0
+pkgdesc="Scala 2.12.2"
+url="http://scala-lang.org"
+arch="all"
+license="BSD3"
+
+# bash          : needed for scripts (e.g. scala)
+# openjdk8-jre  : needed for execution 
+depends="bash openjdk8-jre"
```

Comment:
Is the JRE dependency not inferred by abuild automatically? It should do dynamic library analysis on the final executables, but I'm not sure how Scala works. The `bash` dependency reason doesn't need to be documented so thoroughly.